code/datums/ruins/icemoon.dm

```diff
@@ -167,7 +167,7 @@
 	suffix = "icemoon_underground_frozen_comms.dmm"
 
 /datum/map_template/ruin/icemoon/underground/comms_agent
-	name = "Icemoon Listening Post"
+	name = "Ice-Ruin Icemoon Listening Post"
```

Comment:
```suggestion
	name = "Ice-Ruin Listening Post"
```

Reply:
Yeah I was thinking it looked a little awkward.